src/UI/Internal/Utils/Element.elm

```diff
@@ -24,3 +24,11 @@ ellipsisAttrs lineHeightSize =
     ]
         |> List.map tuplesToStyles
         |> (::) Element.clip
+        |> (::) (title titleContent)
+
+
+title : String -> Attribute msg
+title value =
+    value
+        |> HtmlAttrs.title
+        |> Element.htmlAttribute
```

Comment:
Asking as a doubt, do you guys have preference of avoiding point-free style functions for some reason?

Reply:
Jouderian usually finds them less readable, not on every case, but when it finds he comments on reviews.
I end up adopting on not using them, except on anonymous lambdas.

Reply:
I strongly prefer to be explicit with arguments. It is easier to read by devs, but old and new, and don't require us to keep in mind which parameters are supposed to be there.
Also, is the responsibility of the compiler to detect this cases and optimize them accordingly. Write first for devs, then for compilers.